src/client/Health/NodeView.jsx

```diff
@@ -76,29 +283,98 @@ const NodeView = ({ status, onRestart, onShutdown }) => {
       type: 'input',
       sourcePosition: 'top',
       style: {
-        background: colors.background[dataSource.enabled ? 'enabled' : 'disabled'],
-        border: colors.border[dataSource.enabled ? 'enabled' : 'disabled'],
-        width: '140px',
+        background: colors.background[dataSource.enabled ? 'disabled' : 'disabled'],
+        border: colors.border[dataSource.enabled ? 'disabled' : 'disabled'],
+        width: '170px',
+        height: '25%',
       },
       data: {
         label: (
-          <div id={`south-${index}`} className={`oi-box tight text-${dataSource.enabled ? 'success' : 'muted'}`}>
-            <Link to={`/south/${dataSource.dataSourceId}`}>
-              <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
-                {dataSource.dataSourceId}
+          <div className="box-container">
+            <div className="icon-container">
+              <div className="icon-left">
+                <img src={`${ProtocolSchemas[dataSource.protocol].image}`} alt="logo" height="20px" />
               </div>
-              <UncontrolledTooltip placement="top" target={`south-${index}`} innerClassName="oi-pop">
-                {dataSource.dataSourceId}
-              </UncontrolledTooltip>
-              <div>{`(${dataSource.protocol})`}</div>
-            </Link>
-            <PointsButton dataSource={dataSource} />
+              <div className="icon-center flex-grow">
+                {`${dataSource.protocol}`}
+              </div>
+              <div className={`icon-right ${dataSource.enabled ? 'icon-status-enabled' : 'icon-status-disabled'}`} />
+            </div>
+
+            <div id={`south-${index}`} className={`oi-box tight text-${dataSource.enabled ? 'muted' : 'muted'}`}>
+              <Link to={`/south/${dataSource.dataSourceId}`}>
+                <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                  {dataSource.dataSourceId}
+                </div>
+                <UncontrolledTooltip placement="top" target={`south-${index}`} innerClassName="oi-pop">
+                  {dataSource.dataSourceId}
+                </UncontrolledTooltip>
+              </Link>
+              <PointsButton dataSource={dataSource} />
+            </div>
+
+            <div className="icon-settings">
+              <FontAwesomeIcon icon={faEllipsisV} id="SouthOption" />
+              <UncontrolledPopover trigger="legacy" placement="bottom" target="SouthOption">
+                <PopoverHeader> South Options</PopoverHeader>
```

Comment:
I wonder if a header is necessary here. Anyway, the title is not adapted : it is not South Options but rather actions, or settings